refinery/tool_manager/models.py

```diff
@@ -485,17 +489,20 @@ def _check_max_running_containers(self):
         if len(self._django_docker_client.list()) >= max_containers:
             raise VisualizationToolError('Max containers')
 
-    def _get_detailed_input_nodes_dict(self):
+    def _get_detailed_nodes_dict(self, fetch_all_nodes=False):
```

Comment:
It's probably best to avoid flag arguments in favor of separate functions
https://softwareengineering.stackexchange.com/questions/147977/is-it-wrong-to-use-a-boolean-parameter-to-determine-behavior
https://martinfowler.com/bliki/FlagArgument.html